store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -61,7 +61,6 @@
 import org.apache.rocketmq.store.logfile.MappedFile;
 import org.apache.rocketmq.store.util.LibC;
 import org.rocksdb.RocksDBException;
-
```

Comment:
maybe no need to reformat the import in this file and `MessageConfig.java`

Reply:
done